cmd/yggd/grpc.go

```diff
@@ -113,6 +116,33 @@ func (d *dispatcher) Send(ctx context.Context, r *pb.Data) (*pb.Receipt, error)
 	return &pb.Receipt{}, nil
 }
 
+func (d *dispatcher) sendEvents() {
+	for data := range d.eventsQ {
+		d.RLock()
+		w, prs := d.workers[data.Directive]
+		d.RUnlock()
+		if !prs {
+			log.Warnf("cannot route message to directive: %v", data.Directive)
+			continue
+		}
+		f := func(worker worker, data *pb.APIResponse) {
+			conn, err := grpc.Dial("unix:"+w.addr, grpc.WithInsecure())
+			if err != nil {
+				log.Errorf("cannot dial socket: %v", err)
```

Comment:
Should we retry here?

Reply:
I do not think so, workers can die at any moment, no? missing an event is not also critical here.